msteams-platform/messaging-extensions/build-api-based-message-extension.md

```diff
@@ -957,6 +953,20 @@ After the API-based message extension gets a request header with token, perform
   > [!NOTE]
   > The API receives a Microsoft Entra token with the scope set to `access_as_user` as registered in the Azure portal. However, the token isn't authorized to call any other downstream APIs, such as Microsoft Graph.
 
+For more information on how to validate an access token, see [Validate tokens](/azure/active-directory/develop/access-tokens#validate-tokens).
+
+There are several libraries to validate JSON Web Token (JWT). For basic validation, ensure that you check the following:
```

Comment:
```suggestion
There are several libraries to validate JSON Web Tokens (JWT). For basic validation, ensure that you check the following:
```

Reply:
@Benjiiim, if there is just one JSON Web Token, change it to "to validate a JSON Web Token (JWT)".